jung-api/src/main/java/edu/uci/ics/jung/graph/DelegateCTree.java

```diff
@@ -178,7 +183,7 @@ private void setDepth(N node, N parent) {
       height = Optional.of(0);
     } else {
       depths.putIfAbsent(parent, 0);
-      int nodeDepth = Math.max(depths.get(parent) + 1, height.orElseThrow(AssertionError::new));
+      int nodeDepth = Math.max(depths.get(parent) + 1, height.orElse(0));
```

Comment:
If convenient, please merge in the changes you made in #202 (now committed to the repo) to this PR, so that we can focus this change down a bit.

Reply:
When I run `git fetch origin && git merge origin/master`, it tells me the following:

```
C:\Users\Jonathan\dev\Java\IntelliJ Projects\jung>git fetch origin && git merge origin/master
Already up to date.
```

Have I misunderstood what you're asking me to do here?

Reply:
Okay, I think it's done now.

Reply:
Yeah, sorry, I kind of said that backwards.  You correctly inferred what I was asking for, thanks.  :)